### PR TITLE
#167528341 Add access control to App routes

### PR DESCRIPTION
**Pull request by CryceTruly**
## Description ##
- Add access control to routes based on login status

## Type of change ##
Please select the relevant option
- [x] Bug fix(a non-breaking change which fixes an issue)
- [x] New feature(a non-breaking change which adds functionality)
- [ ] Breaking change(fix of a feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes
- [ ] End to end test
- [ ] Integration test
- [x] unit test

## Checklist: ##
- [x] My code follows the style guidelines of this project
- [x] I have linted my code prior to submission
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes

## Related Pivotal Tracker stories ##
[#167528341](https://www.pivotaltracker.com/story/show/167528341)

### CODE REVIEW

**Review by patrickf949 — COMMENTED** [review]
Great work sofar @CryceTruly 
The route protection on this seems solid.
However, there are 2 navbars upon login
Please work on this.

**Review by Victoradukwu — COMMENTED** [review]
Hey, @CryceTruly This is really great work. Kindly resolve the concern raised by @patrickf949. It is still there.

**Comment by CryceTruly** [comment]
> Hey, @CryceTruly This is really great work. Kindly resolve the concern raised by @patrickf949. It is still there.

Thanks, @Victoradukwu, I have worked on that.

**Comment by CryceTruly** [comment]
> Great work sofar @CryceTruly
> The route protection on this seems solid.
> However, there are 2 navbars upon login
> Please work on this.

Thanks, @patrickf949, I have worked on that.

**Review by JamesMudidi — COMMENTED** [review]
@CryceTruly, This is awesome work.
I noticed that we have two footers on the landing page.
could this be worked on? Thanks

**Comment by CryceTruly** [comment]
> @CryceTruly, This is awesome work.
> I noticed that we have two footers on the landing page.
> could this be worked on? Thanks

Thanks, James, I have worked on this.

**Review by patrickf949 — APPROVED** [review]
LGTM

**Review by Victoradukwu — APPROVED** [review]
LGTM

File: src/app/guards/no-auth.guard.spec.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+			imports: [ RouterTestingModule ],
+			providers: [ NoAuthGuard, { provide: { AuthService, useValue: MockAuthService } } ]
+		});
+		router = new MockRouter();
+		MockAuthService = jasmine.createSpyObj([ 'isLoggedIn' ]);
+		authGuard = new NoAuthGuard(MockAuthService, router);
+	});
+
+	it(
+		'should ...',
```
Comment (JamesMudidi) [inline src/app/guards/no-auth.guard.spec.ts]: Hey @CryceTruly, I am not sure what this is dong here.
can you shine some more light to it

**Review by JamesMudidi — CHANGES_REQUESTED** [review]

**Review by JamesMudidi — APPROVED** [review]
LGTM

**Review by bl4ck4ndbr0wn — APPROVED** [review]
Great Job.
LGTM

**Comment by bl4ck4ndbr0wn** [comment]
Try to resolve the merge conflicts.

**Review by malep2007 — APPROVED** [review]

**Pull request merged**